core/src/test/java/org/hibernate/ogm/backendtck/embeddable/ElementCollectionFromJPQLQueryTest.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.backendtck.embeddable;
+
+import static org.fest.assertions.Assertions.assertThat;
+
+import java.util.List;
+
+import org.hibernate.ogm.utils.OgmTestCase;
+import org.hibernate.ogm.utils.TestForIssue;
+import org.junit.After;
+import org.junit.Test;
+
+/**
+ * Test that the collection of elements is loaded with the entity when it's the result of a JPQL query using JPA.
```

Comment:
What you think of: `Tests that {@link ElementCollection} is loaded within the entity when it's the result of a JPQL query using Hibernate.`